packages/orama/package.json

```diff
@@ -20,11 +20,6 @@
       "types": "./dist/components.d.ts",
       "import": "./dist/components.js",
       "require": "./dist/cjs/components.cjs"
-    },
-    "./stemmers/*": {
```

Comment:
That means that we're still exporting ALL the stemmers together right? Can you please generate a single export per each stemmer?